jobs/build/coreos-installer_sync/Jenkinsfile

```diff
@@ -83,22 +98,47 @@ node {
         )
     }
 
-    stage("Sync to mirror") {
-        def mirror = "use-mirror-upload.ops.rhcloud.com"
-        def dir = "/srv/pub/openshift-v4/x86_64/clients/coreos-installer"
+    stage("calculate shasum") {
+        commonlib.shell(
+            script: "cd ${workdir} && sha256sum * > sha256sum.txt",
+        )
+    }
 
+    stage("Sync to mirror") {
         if (params.DRY_RUN) {
             commonlib.shell(
                 script: [
                     "echo 'Would have synced the following to ${mirror}:'",
                     "tree ${workdir}/${params.VERSION}",
                 ].join('\n')
             )
+        } else {
+            sh "tree ${workdir} && cat ${workdir}/sha256sum.txt"
+            commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/${params.VERSION}/")
+            commonlib.syncDirToS3Mirror("${workdir}/${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/coreos-installer/latest/")
+        }
+    }
+
+    stage("sign artifacts") {
+        if (params.DRY_RUN) {
+            echo "Would have signed artifacts"
             return
         }
+        release.signArtifacts(
+            name: params.VERSION,
+            signature_name: "signature-1",
+            dry_run: params.DRY_RUN,
+            env: params.ENV,
+            key_name: params.KEY_NAME,
```

Comment:
I think parameters `ENV` and `KEY_NAME` need to be removed. They shouldn't be specified when running the job.